### PR TITLE
#12901: Remove extraneous `value: int` parameter from `ttnn.bitwise_not`

### LINKED ISSUE #12901: Extraneous `value: int` parameter from `ttnn.bitwise_not`
Bitwise NOT simply flips all bits.  It uses no additional value.

This issue blocks tenstorrent/pytorch2.0_ttnn#236.

### PR DESCRIPTION
**Pull request by jdh8**
### Ticket
- Resolves #12901
- Blocks tenstorrent/pytorch2.0_ttnn#236

### Problem description
Bitwise NOT simply flips all bits.  It uses no additional value.

### What's changed
Remove extraneous `value: int` parameter from `ttnn.bitwise_not`

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] Device performance regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by ayerofieiev-tt — APPROVED** [review]

File: tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py [hunk 1]
```diff
@@ -862,7 +861,7 @@ def eltwise_unary_rdiv_trunc(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttnn.rdiv(t0, value, round_mode="trunc", memory_config=output_mem_config)
+    t1 = ttnn.rdiv(t0, round_mode="trunc", memory_config=output_mem_config)
```
Comment (eyonland) [inline tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py:865]: Why is rdiv being updated here?   What does this have to do with bitwise_not?

Comment (jdh8) [reply tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py]: Oops thanks! I mistook that. Now I just patched on the intended spot.

**Comment by ayerofieiev-tt** [comment]
@jdh8 this also requires bindings and tests update.

File: tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py [hunk 2]
```diff
@@ -898,7 +897,7 @@ def eltwise_bitwise_not(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttnn.bitwise_not(t0, value, memory_config=output_mem_config, queue_id=0)
+    t1 = ttnn.bitwise_not(t0, memory_config=output_mem_config, queue_id=0)
```
Comment (jdh8) [inline tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py:900]: Besides this, is there any test that need to update?  BTW, how do I run tests locally?

**Comment by jdh8** [comment]
How do I run tests on a LoudBox?  How do I test within `tt-metal` (without involving the compiler)?

**Comment by jdh8** [comment]
This PR was covered by #13579
